test/parallel/test-http-writable-true-after-close.js

```diff
@@ -4,16 +4,13 @@ const common = require('../common');
 const assert = require('assert');
 const { get, createServer } = require('http');
 
-// res.writable should not be set to false after it has finished sending
-// Ref: https://github.com/nodejs/node/issues/15029
```

Comment:
Please read through the issue to make sure you agree with the consequences of this change.